discovery-infra/test_infra/helper_classes/infra_env.py

```diff
@@ -124,4 +135,15 @@ def patch_discovery_ignition(self, ignition_info: str) -> str:
         self.api_client.patch_discovery_ignition(infra_env_id=self.id, ignition_info=ignition_info)
 
     def get_details(self) -> models.infra_env.InfraEnv:
-        return self.api_client.get_infra_env(infra_env_id=self.id)
\ No newline at end of file
+        return self.api_client.get_infra_env(infra_env_id=self.id)
+
+    def update_proxy(self, proxy: Proxy) -> None:
+        self.update_config(proxy=deepcopy(proxy))
+        infra_env_update_params = models.InfraEnvUpdateParams(
+                proxy=models.Proxy(
+                    https_proxy=self._config.proxy.https_proxy if self._config.proxy.https_proxy else None,
+                    http_proxy=self._config.proxy.http_proxy if self._config.proxy.http_proxy else None,
+                    no_proxy=self._config.proxy.no_proxy if self._config.proxy.no_proxy else None
+                )
+        )
+        self.api_client.update_infra_env(infra_env_id=self.id, infra_env_update_params=infra_env_update_params)
```

Comment:
can you add a blank line at the end of the file?

Reply:
done